modules/lmdbbackend/lmdbbackend.cc

```diff
@@ -3149,9 +3163,72 @@ string LMDBBackend::directBackendCmd(const string& query)
     }
   }
 
+  if (cmd == "lookup") {
```

Comment:
compared to DNSBackend method naming, what happens here is much closer to `list` than to `lookup`. Perhaps just rename it to `list`?